WindSpigot-Server/src/main/java/ga/windpvp/windspigot/knockback/KnockbackConfig.java

```diff
@@ -68,11 +68,25 @@ public static void init(File configFile) {
 			windpvpProfile.save(true);
 		}
 		
-		if (!keys.contains("vanilla")) {
-			final KnockbackProfile defaultProfile = new CraftKnockbackProfile("vanilla"); 
-			// Generated profiles are based on kohi kb values, not vanilla, so we set the values here
-			defaultProfile.setFrictionHorizontal(1.0);
-			defaultProfile.setFrictionVertical(1.0);
+		if (!keys.contains("kohi")) {
+			final KnockbackProfile defaultProfile = new CraftKnockbackProfile("kohi"); 
+
+			defaultProfile.setHorizontal(0.35);
+                        defaultProfile.setRodHorizontal(0.35);
+                        defaultProfile.setArrowHorizontal(0.35);
+                        defaultProfile.setPearlHorizontal(0.35);
+                        defaultProfile.setSnowballHorizontal(0.35);
+                        defaultProfile.setEggHorizontal(0.35);
+                        defaultProfile.setAddHotizontal(0.425);
```

Comment:
```suggestion
                        defaultProfile.setExtraHorizontal(0.425);
```

Reply:
It’s addHorizontal, I just spelt it wrong
```suggestion
                        defaultProfile.setAddHorizontal(0.425);
```

Reply:
Add horizontal is different from extra horizontal, add horizontal is added to knockback, and extra horizontal is the knockback multiplier for players' knockback. Add horizontal is an option added by WindSpigot, so did you mean to use the other method?

Reply:
I used addHorizontal/Vertical because I didn’t see extraHorizontal/Vertical and I thought it was renamed :P